AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -967,7 +967,7 @@ - (NSArray *)nodesAtIndexPaths:(NSArray *)indexPaths
 - (NSArray *)completedNodes
 {
   ASDisplayNodeAssertMainThread();
-  return _externalCompletedNodes != nil ? _externalCompletedNodes : _completedNodes[ASDataControllerRowNodeKind];
+  return _externalCompletedNodes ?: _completedNodes[ASDataControllerRowNodeKind];
```

Comment:
In a followup, could you add at least one space between ? and :  -- small ask but this is what other ternarys do in the project and I feel it is easier to read the gap.